src/Microsoft.SqlTools.ServiceLayer/SchemaCompare/SchemaCompareIncludeExcludeNodeOperation.cs

```diff
@@ -97,10 +138,13 @@ private bool IsEqual(SchemaDifference difference, DiffEntry diffEntry)
             System.Reflection.PropertyInfo[] properties = diffEntry.GetType().GetProperties();
             foreach (var prop in properties)
             {
-                result = result &&
-                    ((prop.GetValue(diffEntry) == null &&
-                    prop.GetValue(entryFromDifference) == null) ||
-                    prop.GetValue(diffEntry).SafeToString().Equals(prop.GetValue(entryFromDifference).SafeToString()));
+                if (prop.Name != "Included")
```

Comment:
good catch if I understand this correctly. Can you please add a comment here too.

Reply:
added